web/src/hooks/useRegistryApi.tsx

```diff
@@ -50,33 +50,43 @@ export default function useRegistryApi() {
   const getMinistry = async (): Promise<AxiosResponse<any>> => {
     if (!axiosInstance.current) {
       throw new Error(errorMsg);
-    } else {
-      return axiosInstance.current.get('ministry');
     }
+    return axiosInstance.current.get('ministry');
+  }
+
+  const getQuota = async (): Promise<AxiosResponse<any>> => {
```

Comment:
Looks like `getQuota` api call is not being used anywhere on the web component. Is there a reason you declared this async function?

Reply:
Although it isn't used, I thought it would be nice to have it in there in case it would be needed in the future.
Would you say it's bad practice? Should I remove it?